README.rst

```diff
@@ -172,6 +172,8 @@ Documentation is available `here <http://nidaqmx-python.readthedocs.io>`_.
 Additional Documentation
 ========================
 
+Refer to the `Getting Started with NI-DAQmx <https://www.ni.com/en-us/support/documentation/supplemental/06/getting-started-with-ni-daqmx--main-page.html>`_ page for more detailed documentation of the C and labview interfaces and higher level concepts such as Tasks. The python API follows the C API names relatilvey closely.
```

Comment:
Capitalize LabVIEW and Python.

Also maybe reword the first sentence to something like. 
... for more detailed documentation of the C and LabVIEW interfaces along with higher level API concepts such as Tasks.

Reply:
done